src/Table/Footer.tsx

```diff
@@ -60,15 +61,27 @@ export default function Footer({
     return null;
   }
 
-  // Find the widest possible number range <= `VISIBLE_PAGE_RANGE_SIZE`.
-  const pageNumberPadding = (VISIBLE_PAGE_RANGE_SIZE - 1) / 2;
+  // As a safeguard for when the typescript PageSizeNumber type is not enforced (jsx files),
+  // check if the visiblePageRangeSize is not a PageSizeNumber (even, < 1, > 11),
```

Comment:
I briefly interpreted `(even, < 1, > 11)` as the description of a valid PageSizeNumber. To avoid confusion here, let's have this comment use affirmative language:

```
check that visiblePageRangeSize is a valid PageSizeNumber (1, 3, 5, 7, 9),
and if not, set it to the default value.
```